gsoc_dashboard/phase1_dashboard_prototype.md

```diff
@@ -0,0 +1,36 @@
+# Creation of Dashboard and integration of SAST tool for Automatic generation of assessment scores [GSOC IDEA]
```

Comment:
Don't use "_" in the file name separate with "-".

Reply:
ok thank you